setup.py

```diff
@@ -1,9 +1,70 @@
-from setuptools import setup
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import os
+import sys
+
+from importlib.machinery import SourceFileLoader
+try:
+    from setuptools import setup
+except ImportError:
+    from distutils.core import setup
+
+# avoid loading the package before requirements are installed:
+version = SourceFileLoader('version', 'apicli/version.py').load_module()
+
+__version__ = str(version.VERSION)
+
+if sys.argv[-1] == 'tag':
+    os.system("git tag -a %s -m 'version %s'" % (__version__, __version__))
+    os.system("git push --tags")
+    sys.exit()
+
+if sys.argv[-1] == 'publish':
+    os.system('python setup.py sdist upload')
+    os.system('python setup.py bdist_wheel upload')
+    sys.exit()
+
+if sys.argv[-1] == 'test':
+    os.system('py.test')
+    sys.exit()
+
+readme = open('README.rst').read()
+history = open('HISTORY.rst').read().replace('.. :changelog:', '')
 
 setup(
     name='apicli',
-    version='0.1',
-    py_modules=['restcli'],
+    version=__version__,
+    description='An API client library and CLI written in Python. Postman for terminal lovers!',
+    long_description=readme + '\n\n' + history,
+    author='Dustin Rohde',
+    author_email='dustin@gooee.com',
+    url='https://github.com/dustinrohde/restcli',
+    include_package_data=True,
+    license="Apache",
```

Comment:
can you change this to `license="Apache License, v2.0"`?
Or is `license="Apache"` more standard?